libensemble/tests/unit_tests/test_executor.py

```diff
@@ -781,6 +812,7 @@ def test_non_existent_app():
         assert 0
 
 
+@pytest.mark.extra
```

Comment:
Is putting this decorator before every function the "right way" to do this?

Reply:
We made it ourselves, I put it before tests that fail on windows so they don't get run. Most of the executor tests would be very difficult, if not impossible, to get running on windows since `mpicc` isnt available.

The decorator is produced by pytest based on the settings in `unit_tests/conftest.py`